lib/email_api/email_template_schemas.yaml

```diff
@@ -33,6 +33,13 @@ shutoff_error:
   html_filepath: "html/shutoff_error.html.j2"
   plaintext_filepath: "plaintext/shutoff_error.txt.j2"
 
+shutoff_vm:
+  schema:
+    username: null
+    shutoff_table: "No Shutoff VMs"
+  html_filepath: "html/shutoff_vm.html.j2"
+  plaintext_filepath: "plaintext/shutoff_vm.txt.j2"
+
```

Comment:
You need to add an `email_template_schema` entry here pointing to your `errored_vm` templates - copy the `shutoff_vm` entry